addons/hr_contract/models/hr_contract.py

```diff
@@ -177,6 +177,16 @@ def _check_dates(self):
                     contract=contract.name, start=contract.date_start, end=contract.date_end,
                 ))
 
+    def _get_normalized_wage(self):
+        # This method is overriden in hr_payroll as without that module nothing allows to know
+        # at which frequency is paid the employee
+        wage = self._get_contract_wage()
+        # without payroll installed, we suppose that the employee with a specific schedule has a monthly salary
+        if self.resource_calendar_id:
+            return wage * 12 / 52 / self.resource_calendar_id.hours_per_week
```

Comment:
There's not exactly 52 weeks in a year, and there's leap years, but I guess we don't care here?

Reply:
No I guess it's not necessary to be so specific, unless there is a method to do it :)